NitroxModel/Helper/PhysicsHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace NitroxModel.Helper;
+
+public class PhysicsHelper
```

Comment:
```suggestion
public static class SoundHelper
```

Physics is too generic and volume can mean 3D space.